cmd/init.go

```diff
@@ -0,0 +1,160 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"path"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/fsext"
+)
+
+const defaultNewScriptName = "script.js"
+
+//nolint:gochecknoglobals
+var (
+	errFileExists            = errors.New("file already exists")
+	defaultNewScriptTemplate = template.Must(template.New("init").Parse(`import http from 'k6/http';
```

Comment:
```suggestion

const (
	defaultNewScriptTemplate = `import http from 'k6/http';
```
The error is not used anymore so we can remove it. I think we can move the string as a constant and create directly the template on the fly without allocating it for every command that has the benefit of not requiring the `nolint` directive.

Reply:
I usually opt for parsing templates at the very top because it has pretty small footprint — parsed templates tend to be singletones — while providing the benefit of validating templates without running the command, which makes testing easier.

Reply:
>it has pretty small footprint

It is secondary to me, I care more about the fact we are defining a global var. It sounds like we don't have a real need - as we are not using it. It seems there isn't any test using it.

Reply:
The one that asserts on the content of generated file does :) It just checks the major parts, not the full text, since it’s going to be a pain to maintain it otherwise.